tracee-rules/signature.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/aquasecurity/tracee/tracee-rules/types"
 )
 
-func getSignatures(rulesDir string, rules []string) ([]types.Signature, error) {
+func GetSignatures(rulesDir string, rules []string) ([]types.Signature, error) {
```

Comment:
I can swing either way - for with or without this change to export this function. I just thought it'd be easier in the future to use as an abstraction.

Reply:
fine with me, but how can someone practically import this?

Reply:
I mean import main, from another package in this module?

Reply:
Actually your second comment made me think again - yes, e.g. an integration test. But I think it's better to do that when the need arises. I'll remove this for now.